src/app/[locale]/(logged-out)/search/components/Search/Search.tsx

```diff
@@ -980,35 +947,144 @@ const Search = ({ filters, cohortDiscovery }: SearchProps) => {
+                                            aria-live="polite">
+                                            {data &&
+                                                data.path?.includes(
+                                                    queryParams.type
+                                                ) &&
+                                                getXofX()}
+                                        </Box>
+                                    </Box>
+                                    {!isMobile && !isTabletOrLaptop && (
+                                        <Box sx={{ display: "flex", gap: 2 }}>
+                                            <Box sx={{ p: 0 }}>
+                                                <Sort
+                                                    sortName={SORT_FIELD}
+                                                    defaultValue={
+                                                        queryParams.sort
+                                                    }
+                                                    submitAction={onSortChange}
+                                                    sortOptions={getSortOptions()}
+                                                    iconised={false}
+                                                />
+                                            </Box>
+                                            {!excludedDownloadSearchCategories.includes(
+                                                queryParams.type
+                                            ) && (
+                                                <Button
+                                                    onClick={() =>
+                                                        !isDownloading &&
+                                                        downloadSearchResults()
+                                                    }
+                                                    variant="contained"
+                                                    color="greyCustom"
+                                                    startIcon={
+                                                        <DownloadIcon color="primary" />
+                                                    }
+                                                    disabled={
+                                                        isDownloading ||
+                                                        !data?.list?.length
+                                                    }>
+                                                    {t("downloadResults")}
+                                                </Button>
+                                            )}
+                                            <Button
+                                                variant="contained"
+                                                color="greyCustom"
+                                                disabled={hasNotSearched()}
```

Comment:
seems strange to call a function like this as will only be called when there is a rerender, unless this function is a useCallback